api/configs/env.go

```diff
@@ -15,3 +16,18 @@ func EnvMongoURI() string {
 
 	return os.Getenv("MONGODB_URI")
 }
+
+func EnvLimit() int64 {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file: %v", err)
+	}
+
+	limit, err := strconv.ParseInt(os.Getenv("LIMIT"), 10, 64)
```

Comment:
Should this be an argument or an environment variable?

Reply:
Environment variable I think, what would be the benefit of making it an argument?